backup/Dockerfile

```diff
@@ -36,4 +39,10 @@ ENV SCHEDULE **None**
 ADD run.sh run.sh
 ADD backup.sh backup.sh
 
+# Run as an arbitrary non-root user for security reasons.
+ENV USER 65532
+RUN adduser -D $USER
+RUN chown --recursive $USER $WORKDIR
+USER $USER
```

Comment:
Is this a best practice? I thought that the `USER nonroot:nonroot` is what people use to do. I would leave it up to docker to assign some non-root user.

Reply:
UIDs, like ports or (any fixed size namespace) have some intricacies, system has good documentation on them: https://systemd.io/UIDS-GIDS/

but of course this is mostly specific to systemd, which no sane person would use in a container 😅 . It seems to me that it doesn't really matter what user id we are using, if it's BIG enough.

I just asked to use this 65532, which we use elsewhere too in our custom images. I don't know, I am bit afraid of user_ids, and having to remap them to host users when running rootless podman (in case we ditch docker for good)

Reply:
Ok, thank you for more insight. So, it looks like that we should either use the same UID everywhere or to split our pods with UIDs according to whether they are supposed to share something (e.g. files, as @dszakallas suggested). Using random UIDs does not make sense then.

Reply:
For this particular image, I would advise to use separate UID, since this is not supposed to share anything with any other pod, I believe (correct me please). Having separate UID can then give us another layer of security, which can be then used as a mitigation control for this image (there were recently CVE issues with this image, we spent time on it, although this is definitely not the most important image in CP).

Reply:
Alrighty, reverted back to a different ID.